pg_lakehouse/benchmarks/README.md

```diff
@@ -19,10 +19,10 @@ brew install docker postgresql
 
 ## Running the Benchmarks
 
-You can then run the benchmarks via:
```

Comment:
This file is out of date. We no longer need Docker, or even psql. The dependencies/steps are same as in cargo-paradedb, meaning:

Pgrx, rust, a started Postgres database (likely via cargo pgrx start or cargo pgrx run), etc. 

I propose we remove the `pg_lakehouse/benchmarks/` folder altogether, and update the links in top-level benchmarks/ to reference cargo-paradedb, and add instructions for pg_lakehouse benchmarking to the `cargo-paradedb` README